Sentry.xcworkspace/contents.xcworkspacedata

```diff
@@ -34,9 +34,6 @@
       <FileRef
          location = "group:Samples/iOS-Swift/iOS-Swift.xcodeproj">
       </FileRef>
-      <FileRef
```

Comment:
`h`: Why did you remove this project?

Reply:
hmm, i removed them from the workspace, but i suppose we could leave the references in place.

Reply:
When I started to maintain this SDK I assumed it's best to use the workspace to have access to all relevant files. That's my main reason, but I assume the CLI usage would also have to be changed.

> i'll make this change 👍🏻

Thank you

Reply:
The CLI flow could even be made simpler. Instead of a separate make task per project as I am currently proceeding, I can just make it regenerate all the projects in one go.

Worth noting though, that if we ditch the workspace, we can still generate projects that have access to all the relevant files.

Reply:
This is now done, for this and all subsequent PRs, there is now just the one `make xcode` task, and the projects stay linked in the xcworkspace so they can be used there.